### ISSUE TITLE
If anyone trained  this models,Please post librispeech WER and/or timit PER.

### ISSUE THREAD

**Issue by shiva1393** [Alexander-H-Liu/End-to-end-ASR-Pytorch#38]

**Comment by reflectionie** [comment]
26.9 PER in test set, but I actually find a problem in dev set: PER is so high(greater than 1), and I'm trying to change my configration.
